recipes/imgui/all/test_package/CMakeLists.txt

```diff
@@ -4,5 +4,7 @@ project(test_package)
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
 conan_basic_setup()
```

Comment:
To avoid doing `include_directories`
```suggestion
conan_basic_setup(TARGETS)
```

Reply:
Why? We are not using targets from `cmake` generator.

Reply:
`conan_basic_setup` will not run `include_directories(${CONAN_INCLUDES})` when giving the `TARGETS` argument.